taskt/Core/Automation/Commands/API Commands/ExecuteRESTAPICommand.cs

```diff
@@ -1,356 +1,333 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using RestSharp;
+using System;
 using System.Collections.Generic;
+using System.Data;
 using System.Linq;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using RestSharp;
-using System.Data;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
 
     [Serializable]
-    [Attributes.ClassAttributes.Group("API Commands")]
-    [Attributes.ClassAttributes.Description("This command allows user to call REST API.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to call a REST API with specific HTTP method")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements 'RestRequest' to call a REST API Endpoint with specific HTTP 'Method'")]
+    [Group("API Commands")]
+    [Description("This command allows user to call a REST API with a specific HTTP method.")]
+
     public class ExecuteRESTAPICommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please enter the Base URL (ex. http://mysite.com)")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Define any API endpoint which contains the full URL.")]
-        [Attributes.PropertyAttributes.SampleUsage("**https://example.com** or **{vMyUrl}**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Base URL")]
+        [InputSpecification("Provide the base URL of the API.")]
+        [SampleUsage("https://example.com or {vMyUrl}")]
```

Comment:
Please use || (double pipes) to separate two or more sample usages.

Reply:
Thanks. Forgot those. Done.